### ISSUE TITLE
uplink 0.9.4 is no longer compatible with Python 2.7.5

### ISSUE THREAD

**Issue by selimt** [prkumar/uplink#217]
**Describe the bug**
0.9.4 changed the way abc was imported from collections. See: https://github.com/prkumar/uplink/pull/201
When Python 2.7.5 imports uplink we get the following error:
```
>>> import uplink
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/uplink/__init__.py", line 4, in <module>
    from uplink import returns, types
  File "/usr/lib/python2.7/site-packages/uplink/returns.py", line 6, in <module>
    from uplink import decorators
  File "/usr/lib/python2.7/site-packages/uplink/decorators.py", line 6, in <module>
    from collections import abc
ImportError: cannot import name abc
```

**To Reproduce**
```
pip install uplink --upgrade
python
>>> import uplink
```

**Expected behavior**
We should be able to use uplink with Python 2.7 since it is still advertised as Python 2.7 compatible.

**Additional context**
I have not had a chance to test this to see if this issue is solved in future Python 2.7 releases but Python 2.7.5 is what we currently use.

**Comment by tirkarthi** [comment]
Seems 2.7 and 3.4 testing have been removed : https://github.com/prkumar/uplink/pull/199

**Comment by Imaclean74** [comment]
Is it better to just deprecate 2.x support at this point ?

**Comment by prkumar** [comment]
I would prefer deprecating 2.x support at this point, considering we are over a year since 2.7 EOL. As more of our dependencies deprecate 2.7, supporting it will become more and more difficult.

@selimt - are you still using 2.7?

**Comment by selimt** [comment]
@prkumar Yes, we are in the process of upgrading all our code to Python 3.7 but currently we still depend on 2.7. If supporting 2.7 is no longer feasible we'll pin our requirement for 2.7 to uplink 0.9.3.
Thanks.

**Comment by prkumar** [comment]
I've merged #223 to add back 2.7 support in the upcoming patch release (i.e., v0.9.5). I'm thinking we should completely remove 2.7 support in the next minor version release (i.e., v0.10.0).

Thanks @shadchin for #223.
